ql/src/java/org/apache/hadoop/hive/ql/ddl/process/show/compactions/ShowCompactionsDesc.java

```diff
@@ -33,8 +33,8 @@
 
   // @formatter:off
   public static final String SCHEMA =
-      "compactionid,dbname,tabname,partname,type,state,hostname,workerid,enqueuetime,starttime,duration,hadoopjobid,errormessage#" +
-      "string:string:string:string:string:string:string:string:string:string:string:string:string";
+      "compactionid,dbname,tabname,partname,type,state,workerhostname,workerid,enqueuetime,starttime,duration,hadoopjobid,errormessage,initiatorhostname,initiatorid#" +
```

Comment:
i would go with just 'workerhost' and 'initiatorhost'

Reply:
fixed